docs/INSTALL.md

```diff
@@ -57,10 +57,8 @@ To pass the information about the bootstrap config repo to Honeydipper daemon, t
 ---
 daemon:
   env:
-    - name: REPO
-      value: git@github.com/example/honeydipper-config.git
-    - name: HONEY_SSH_KEY
-      valueFrom:
+    REPO: git@github.com/example/honeydipper-config.git
+    HONEY_SSH_KEY:
```

Comment:
Can you work on the code changes to support renaming these?

Reply:
This was done in the previous PR #76 in the chart template.  https://github.com/honeydipper/honeydipper/blob/1413aab514018602c0d7099ac8e841ac5cd66097/deployments/helm/honeydipper/templates/daemon-deployment.yaml#L64-L75